plugins/fresco-pd/lsusb.txt

```diff
@@ -55,7 +55,7 @@ Binary Object Store Descriptor:
     bLength                    48
     bDescriptorType            16
     bDevCapabilityType         13
-    iAddtionalInfoURL           4 www.frescologic.com
+    iAdditionalInfoURL          4 www.frescologic.com
```

Comment:
?

Reply:
Looks like libusb needs the same checks :) this is fine for me

Reply:
Added to https://github.com/stars/jsoref/lists/to-check-spelling, but the list doesn't do a great job of shrinking...